dlib/cuda/cuda_dlib.h

```diff
@@ -412,6 +412,62 @@ namespace dlib
 
     // ----------------------------------------------------------------------------------------
 
+        class compute_loss_binary_log_per_pixel
+        {
+            /*!
+                The point of this class is to compute the loss computed by
+                loss_binary_log_per_pixel, but to do so with CUDA.
```

Comment:
Should reference `loss_binary_log_per_pixel_` not `loss_binary_log_per_pixel`

Reply:
Fixed – also at the source where this was copied from.